### ISSUE TITLE
Sessions not implemented?

### ISSUE THREAD

**Issue by NorfairKing** [dmjio/stripe#120]
This seems to be required for all payments , before September 15 2019. Is there any plan to add this? How can I help?

**Comment by dmjio** [comment]
At the time this API was implemented, I don't believe sessions were present.

https://web.archive.org/web/20140625062328/https://stripe.com/docs/api

**Comment by NorfairKing** [comment]
Yes, that makes sense, but on Sept 15 2019, new regulation comes into effect that requires everyone using stripe to use the new API. This includes sessions.

Also, in `EventData`, the `UnknownEventData` constructor should probably contain a `Data.Aeson.Value`.

**Comment by NorfairKing** [comment]
The `EventType` constructor called `UnknownEventType` should probably also include a `Text`.

**Comment by danstn** [comment]
+1

Currently parsing "checkout.session.completed" event manually. Would be great to have Session support :)

**Comment by NorfairKing** [comment]
@danstn Could you help with #122 then?
